types/pseudo-audio-param/index.d.ts

```diff
@@ -0,0 +1,47 @@
+// Type definitions for pseudo-audio-param 1.3
+// Project: https://github.com/mohayonao/pseudo-audio-param/
+// Definitions by: Drew Petersen <https://github.com/kirbysayshi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "pseudo-audio-param" {
```

Comment:
this is the old UMD pattern. The new style uses the syntax `export as namespace PseudoAudioParam`. So the code would look like this:

```ts
export = PseudoAudioParam;
export as namespace PseudoAudioParam;
// rest of code is the same 
interface PseudoAudioParamEvent {
   // etc etc etc...
```

Reply:
Ok, I made the changes requested! Thanks for the sample code. I find UMD typings one of the most difficult to get understand and get right.